docs/metrics/healthcheck/README.md

```diff
@@ -16,8 +16,20 @@ This probe is used to identify situations where the server is not ready to accep
 
 Internally, MinIO readiness probe handler checks for backend is alive and in read quorum then the server returns 200 OK, otherwise 503 Service Unavailable.
 
-Platforms like Kubernetes *do not* forward traffic to a pod until its readiness probe is successful. 
+Platforms like Kubernetes *do not* forward traffic to a pod until its readiness probe is successful.
 
 ### Configuration example
 
 Sample `liveness` and `readiness` probe configuration in a Kubernetes `yaml` file can be found [here](https://github.com/minio/minio/blob/master/docs/orchestration/kubernetes/minio-standalone-deployment.yaml).
+
+### Configure readiness deadline
+Readiness checks need to respond faster in orchestrated environments, to facilitate this you can use the following environment variable before starting MinIO
```

Comment:
Can you say `to respond (by 200 Ok or 503 Service Unavailable) faster`

because otherwise it will be little confusing

Reply:
Its all about response, it doesn't matter what is the HTTP header that we send @vadmeste